src/test/rules/sortImportsRuleTests.ts

```diff
@@ -0,0 +1,237 @@
+/// <reference path='../../../typings/mocha/mocha.d.ts' />
```

Comment:
For the tests lets try to use the new format which uses the rule tester. See this example from another contributor: https://github.com/buzinas/tslint-eslint-rules/blob/master/src/test/rules/spaceInParensRuleTests.ts

Try to also add the output, if there is any. I noticed that on the eslint rule tests they set the output to null: https://github.com/eslint/eslint/blob/master/tests/lib/rules/sort-imports.js, in such case just use the empty string `''` since later on we will turn on the strictNull option on the typescript compiler.

Here is another recent test file: https://github.com/buzinas/tslint-eslint-rules/blob/master/src/test/rules/terArrowBodyStyleRuleTests.ts